### PR TITLE
[chore] Styling of Project Modal (w/o KDMs)

### PR DESCRIPTION
**Pull request by nehaahussain**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Changed all the styling to use Styled Components except for images.
- Adjusted design to match Hifis without KDMs.
- Fixed minor errors in `texts.ts`.
- Installed react feather icons.

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."
Screenshot 2024-10-28 at 1 53 42 PM

Update:
Screenshot 2024-10-29 at 8 54 04 PM

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
- `pnpm dev` then press on one of the markers.

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."
- adjust styling to account for browser size (monique said something abt scrolling through the modal when the browser height is smaller??)
- fix small styling thing when u make the browser full screen, i will do it

CC: @itsliterallymonique

### CODE REVIEW

File: components/ProjectModal/index.tsx [hunk 1]
```diff
@@ -46,39 +47,58 @@ export default function ProjectModal({
...
-        <div>{size}</div>
-        <div style={styles.additionalInfoStyles}>
-          DETAILS
-          <br></br>
-          {additional_information}
-        </div>
-      </Modal>
+      <styles.ModalContent isOpen={openFirst}>
+        <styles.ModalOverlay>
+          <styles.SearchBar>Search</styles.SearchBar>
```
Comment (itsliterallymonique) [inline components/ProjectModal/index.tsx:52]: remove the search bar (this will directly be on the home page)

**Review by itsliterallymonique — CHANGES_REQUESTED** [review]
Thanks -- styling is really nice! Check my comments. Also make it so that the user can still interact with the google maps when the project modal is opened (you could do this in Ethan's latest PR so I think you might have made a change to a component / field)

File: components/ProjectModal/index.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 'use client';
 
 import React, { useEffect, useState } from 'react';
-import Modal from 'react-modal';
+import { FiX, FiZap } from 'react-icons/fi';
 import { queryProjectbyId } from '../../api/supabase/queries/query';
+import * as texts from '../../styles/texts';
 import { Project } from '../../types/schema';
 import * as styles from './styles';
```
Comment (jjstnlee) [inline components/ProjectModal/index.tsx:8]: If possible, I think we should import all of the styled components individually as their names instead of as styles so that we don't have to start everything with "styles.---". It looks kinda hard to read with all of the "styles.---".

Comment (itsliterallymonique) [reply components/ProjectModal/index.tsx:8]: yeah I think in my previous project we also imported all styling individually instead of using styles. or texts. So I agree that we should change it to import individually. E.g. in texts.ts I imported all the fonts as: `import { CoinbaseMono, CoinbaseSans, CoinbaseText } from './fonts';` instead of `import * as fonts from ./fonts`

I will also make this announcement on slack

**Review by itsliterallymonique — APPROVED** [review]
looks really good neha! you can squash and merge it.
